tests/unit/testExpressInterface.js

```diff
@@ -14,54 +14,42 @@
  * limitations under the License.
  */
 
-var test = require('tape');
 var lodash = require('lodash');
+var assert = require('assert');
 var merge = lodash.merge;
 var expressInterface = require('../../src/interfaces/express.js');
 var ErrorMessage = require('../../src/classes/error-message.js');
 var Fuzzer = require('../../utils/fuzzer.js');
 var Configuration = require('../fixtures/configuration.js');
 var createLogger = require('../../src/logger.js');
 
-test(
-  "Given invalid, variable input the express interface handler setup should not throw errors"
-  , function ( t ) {
-
-    var f = new Fuzzer();
-
-    t.doesNotThrow(
-      function ( ) {
-
-        f.fuzzFunctionForTypes(
-          expressInterface
-          , ["object", "object"]
+describe('expressInterface', function () {
+  describe('Exception handling', function () {
+    describe('Given invalid input', function () {
+      it('Should not throw errors', function () {
+        var f = new Fuzzer();
+        assert.doesNotThrow(
+          function () {
+            f.fuzzFunctionForTypes(
+              expressInterface
+              , ["object", "object"]
+            );
+            return;
+          }
         );
-
-        return ;
-      }
-      , undefined
-      , "The express interface handler setup should not throw when given invalid types"
-    );
-
-    t.end();
-  }
-);
-
-test(
-  [
-    "Given valid setup variables from the handler setup, the bound express"
-    , "error handler should produce a determinate error message and callback to"
-    , "the given callback function"
-  ].join(" ")
-  , function ( t ) {
-
+      });
+    });
+  });
+  describe('Intended behaviour', function () {
```

Comment:
The `it` test cases in `sendError`/`nextCb` are guaranteed to pass, since `assert(true)` is a no-op. Also, if I'm running the test and the callbacks aren't called at all, then unless I manually scan the entire console output to see that the `Should callback to the * function` messages are missing, then I would have no idea.